src/community/geofence-server/src/main/java/org/geoserver/geofence/rest/UsersRestController.java

```diff
@@ -0,0 +1,151 @@
+package org.geoserver.geofence.rest;
```

Comment:
headers :)

Not sure where/if you want to move this, I kind of like having all the REST security work in one place.